db/schema.rb

```diff
@@ -0,0 +1,26 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2019_11_11_162455) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "pgcrypto"
+  enable_extension "plpgsql"
+
+  create_table "hearing_results", force: :cascade do |t|
```

Comment:
Open question 💭 

As, if I remember right, we will be grouping the hearings by their Prosecution Case, would make sense to have a `ProsecutionCaseID` indexed column (populated from the JSON values when populating the model) so we can create rails associations between the `ProsecutionCases` and the `HearingResults`?  Or leave it like this, create a JsonB index on the `ProsecutionCaseID` attribute in the JSONB and directly create associationgs with a gem like this: https://github.com/y9v/activerecord-jsonb-associations ?
Or any other options?

Reply:
I think this will be the other way around, because hearings have a one to many relationship with [prosecution cases](https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/master/lib/schemas/global/hearing.json#L21-L28)
We can add a hearing_id on the `prosecution_cases` table I suppose?

Reply:
Been checking/refreshing on the endpoint data views and the json schemas.

From the initial search (that returns a Prosecution Case with its associated hearings) to the Record LAA Reference (over a Prosecution Case) seems the driver for collecting and searching the hearings will be the `ProsecutionCaseID`
Then, when we get the hearing, as you said, seems a hearing can be associated to multiple prosecution cases. 
We definitely need to be able to have an association that allows us for a quick "give me all the stored hearings for this ProsecutionCase".

Maybe a many to [many association](https://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association) is the way to go?